modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -327,47 +294,40 @@ amazon.secretkey=<Secret Key>
 amazon.region=<Region>
 ----
 +
-. Drag an HTTP Listener component onto the canvas and configure the following parameters:
-+
-image::amazon/amazon-ec2-http-props.png[ec2 HTTP configure properties]
-+
-[%header%autowidth.spread]
-|===
-|Parameter |Value
-|Display Name |Listener
-|Extension Configuration | If no HTTP Listener configuration has been created yet, click the plus sign to add a new HTTP Listener Configuration and click OK (Give values "localhost" and "8081" for Host and Port columns).
-|Path |/createVolume
-|===
-+
-. Drag the Amazon EC2 Connector "Create volume" next to the HTTP Listener component.
+. Drag the HTTP Listener component to the canvas, and set *Host* to `localhost` and *Port* to the `8081` value.
+. Drag the Amazon EC2 Connector *Create volume* operation next to the HTTP Listener component.
 . Configure the EC2 connector by adding a new Amazon EC2 Global Element. Click the plus sign next to the Connector Configuration field.
-.. Configure the global element according to the table below:
+. Configure the global element:
 +
-[%header%autowidth.spread]
+[%header,cols="30s,40a,30a"]
 |===
 |Parameter |Description |Value
 |Name |Enter a name for the configuration to reference it. |<Configuration_Name>
```

Comment:
ditto previous edits re. making first item fragment for parallelism and omitting periods in the Description column